src/main/resources/swagger/api-docs.yaml

```diff
@@ -145,31 +144,24 @@ paths:
           required: true
           type: string
       responses:
-        '200':
-          description: Proxy connection successful
+        '202':
+          description: Cluster deletion request accepted
         '404':
           description: Cluster not found
           schema:
             $ref: '#/definitions/ErrorReport'
-        '420':
-          description: Cluster not ready
-          schema:
-            $ref: '#/definitions/ErrorReport'
         '500':
           description: Internal Error
           schema:
             $ref: '#/definitions/ErrorReport'
-      security:
-        - googleoauth:
-            - openid
-            - email
-            - profile
-    delete:
-      summary: Deletes an existing dataproc cluster in the given project
-      description: deletes a dataproc cluster
-      operationId: deleteCluster
+
+  '/api/notebooks/{googleProject}/{clusterName}':
```

Comment:
oh wow this is cool. i was worrying that we have a "get cluster details" and a "view notebook" and they're both at GET, but one is at /cluster and one at /notebooks. neat 👍